lib/filesystem/ResourceID.cpp

```diff
@@ -129,6 +129,7 @@ EResType::Type EResTypeHelper::getTypeFromExtension(std::string extension)
 		{".MSG",   EResType::MASK},
 		{".H3C",   EResType::CAMPAIGN},
 		{".H3M",   EResType::MAP},
+		{".TUT",   EResType::MAP},
```

Comment:
This will also make tutorial show up in map list.
Not needed and not visible in H3 either

Reply:
I introduced workaround to filter tutorial map from scenario list, but yes it's not the best approach